chapters/operatorsandexpressions.tex

```diff
@@ -538,7 +538,8 @@ \subsection{Numeric Functions and Conversion Functions}\label{numeric-functions-
 The ANSI-C style \lstinline!format! string (which cannot be combined with any of the other named arguments) consists of a single conversion specification without the leading \%.
 It shall not contain a length modifier, and shall not use `\lstinline!*!' for width and/or precision.
 For both \lstinline!Real! and \lstinline!Integer! values, the conversion specifiers `\lstinline!f!', `\lstinline!e!', `\lstinline!E!', `\lstinline!g!', `\lstinline!G!' are allowed.
-For \lstinline!Integer! values it is also allowed to use the `\lstinline!d!', `\lstinline!i!', `\lstinline!o!', `\lstinline!x!', `\lstinline!X!', `\lstinline!u!', and `\lstinline!c!' conversion specifiers (for \lstinline!Real! values a tool may round, truncate or use a different format if the integer conversion specifiers are used).
+For \lstinline!Integer! values it is also allowed to use the `\lstinline!d!', `\lstinline!i!', `\lstinline!o!', `\lstinline!x!', `\lstinline!X!', `\lstinline!u!', and `\lstinline!c!' conversion specifiers.
+Using the \lstinline!Integer! conversion specifiers for a \lstinline!Real! value is a deprecated feature without defined result.
```

Comment:
Merely saying 'without defined result' seems too general. I would expect the deprecated variant to keep the previous behavior, i.e. round/truncate or use a different format.

If it is used for printing a result for human users (which seems like a common scenario) then round/truncate or different format all work to some degree.

Reply:
OK now?